lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts

```diff
@@ -63,6 +63,12 @@ export class AttachFileWidgetCloudPage {
         }
     }
 
+    async checkNoFileIsAttached(): Promise<void> {
+        const filesListLocator = 'div[class="adf-file-properties-table"]';
+        const fileItem = this.widget.$(filesListLocator).$('table');
```

Comment:
can we use the TestElement from testing core lib? lib/testing/src/lib/protractor/core/test-element.ts

Reply:
evrywhere